src/v2/ion/uiSchemaTransformer.js

```diff
@@ -154,19 +154,26 @@ const createAuthenticatorOptions = (options = [], authenticators = []) => {
  * @param {IONForm} remeditationForm
  */
 const createUISchema = (transformedResp, remediationForm) => {
-  /* eslint complexity: [2, 14] */
+  /* eslint complexity: [2, 15] */
 
   // For cases where field itself is a form, it has a formname and we are appending the formname to each field.
   // Sort of flat the structure in order to align Courage flatten Model. The flatten structure will be converted
   // back to object hierarchy through `Model.toJSON`.
   // For simplicity we are assuming that when field itself is a form its only one level deep.
   const remediationValue = _.chain(remediationForm.value || [])
     .map(v => {
+      // `v.form` is probably not right structure but `v.value.form`
+      // TODO: clean up after API stablization.
       if (v.form) {
         const inputGroupName = v.name;
         return v.form.value.map(input => {
           return Object.assign({}, input, { name: inputGroupName + '.' + input.name });
         });
+      } else if (v.value && v.value.form) {
```

Comment:
Instead of a check like this, can we not make sure of this in response transformer?

Also, once we start supporting ES2020, this would be a good candidate for optional chaining.

Reply:
> can we not make sure of this in response transformer?

Reply:
I was referencing to       
// `v.form` is probably not right structure but `v.value.form`
Since it looks like a either-or scenario, can we make this decision in responseTransformer and pick either one ?

Reply:
Possibly. Will leave it for later when we got more real data from API integration testing. My hope is to just remove another `if` block.